src/todo/columns.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+
+import defaultTodoColumns from "./config.json";
+import buildColumn from "../refapp-grid/column-builder";
+
+export default function getColumns() {
+  const getDoneActionColumn = () => {
+    return [];
+  };
+
+  const getTodoActionColumn = () => {
+    const fetchEncounterURL = (patientId, encounterId) => {
+      let baseUrl: string = `${location.origin}/openmrs/htmlformentryui/htmlform/viewEncounterWithHtmlForm.page?`;
+
+      return `${baseUrl}patientId=${patientId}&encounter=${encounterId}`;
+    };
+    const getTodoActionButton = todo => {
+      switch (todo.type) {
+        case "PRINT_CONSENT":
+          return (
+            <a
+              title="Basic Service Plan Encounter URL"
+              href={fetchEncounterURL(todo.patient.id, todo.encounterId)}
+            >
+              <button className="task button small-button">Print</button>
```

Comment:
Please use <Trans> for text. So that it will support translation.

Reply:
Done https://github.com/openmrs/openmrs-esm-user-dashboard-widgets/pull/15/commits/35eba98c8075fafe5402a2c0ecbe61498d053f5b#diff-178c5e4eae7d30925dec99b3f1d5e8b8R25